web/src/pages/Consumer/locales/zh-CN.ts

```diff
@@ -20,6 +20,10 @@ export default {
   'page.consumer.form.itemExtraMessage.username': '名称需唯一',
   'page.consumer.notification.warning.enableAuthenticationPlugin': '请启用一种身份认证类插件',
   'page.consumer.username': '名称',
+  'page.consumer.username.required': '请输入消费者名称',
   'page.consumer.updateTime': '更新时间',
-  'page.consumer.list': '消费者列表'
+  'page.consumer.list': '消费者列表',
+  'page.consumer.description': '消费者是路由的消费方，形式包扩开发者、最终用户、API 调用等。创建消费者时，需绑定至少一个认证类插件。',
+  'page.consumer.create-title': '创建消费者',
+  'page.consumer.configure-title': '配置消费者',
```

Comment:
May I ask why not keep the same with service use `page.consumer.configure` here?

Reply:
ya, when I file the new PR, I use `xxx.configure` without title, this i18n will be handled in a new PR.

Reply:
updated